core/scripts/cli/sponsor/currencyUtils.js

```diff
@@ -0,0 +1,18 @@
+const wrapToWeth = async (web3, artifacts, emp, amount) => {
+  const WETH9 = artifacts.require("WETH9");
```

Comment:
nit: why not just put this at the top of the file?

Reply:
Unfortunately we need to use the passed in `artifacts` variable to do the require, so we can't put the require at the top of the file.

Reply:
oh wow, I totally missed that. Understood.